src/components/Article.jsx

```diff
@@ -17,22 +18,25 @@ const Article = () => {
     })
   }, [])
 
-  if (isLoading) return <>loading...</>
+  if (isLoading) return <h1>loading...</h1>
 
   return (
-    <section>
-      <h3>{article.topic}</h3>
-      <h1>{article.title}</h1>
-      <h2>by {article.author}</h2>
-      <h3>{article.created_at}</h3>
-      <img
-        id="articleCover"
-        src={article.article_img_url}
-        alt={`cover image for the article ${article.title}`}
-      />
-      <p>{article.body}</p>
-      <h3>{article.votes} votes</h3>
-    </section>
+    <>
+      <section>
+        <h3>{article.topic}</h3>
+        <h1>{article.title}</h1>
+        <h2>by {article.author}</h2>
+        <h3>{article.created_at}</h3>
```

Comment:
You shouldnt use h1 and h3 etc for sizing. They go in order i.e. h1 for main title
h2 for sub titles - h3 for a sub titles inside the other sub. 

You can change the sizes via css, but should never use them to get different sizes